### PR TITLE
chore(TDKN-200): Add new project daikon-spring-zipkin

### PR DESCRIPTION
**Pull request by fhuaulme**
**What is the problem this Pull Request is trying to solve?**

See https://jira.talendforge.org/browse/TDKN-200
 
**What is the chosen solution to this problem?**
 
See https://jira.talendforge.org/browse/TDKN-200

**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDKN-200
 
**Please check if the Pull Request fulfills these requirements**
- [x] The PR commit message follows our [guidelines](../CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features, coverage should be over 75% in the new code)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Related design / discussions / pages (not in JIRA), if any, are all linked or available in the Pull Request

### CODE REVIEW

File: daikon-spring/daikon-spring-zipkin/src/main/java/org/talend/daikon/zipkin/HistoryZipkinKafkaSenderConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+/**
+ * Configures the zipkin kafka sender for another kafka broker than the functional broker.
+ * spring.zipkin.kafka.bootstrapServers stores the broker dedicated to the infrastructure log events.
+ */
+@Configuration
+@ConditionalOnClass(ByteArraySerializer.class)
+@ConditionalOnMissingBean(Sender.class)
+@ConditionalOnProperty(value = "spring.zipkin.enabled", havingValue = "true")
+public class HistoryZipkinKafkaSenderConfiguration {
```
Comment (afourmi) [inline daikon-spring/daikon-spring-zipkin/src/main/java/org/talend/daikon/zipkin/HistoryZipkinKafkaSenderConfiguration.java]: rename it to ZipkinKafkaSenderConfiguration (History is the name of one TDS module)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/28/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/28/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/29/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/29/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/daikon_pull-request_master/30/) [:star:](https://ci-common.datapwn.com/job/daikon_pull-request_master/30/)

**Pull request merged**